build.gradle

```diff
@@ -52,7 +51,7 @@ buildscript {
         classpath "com.github.ben-manes:gradle-versions-plugin:$gradleVersionsPluginVersion"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
-        classpath 'io.arrow-kt:ank-gradle-plugin:0.7.1'
+        classpath "io.arrow-kt:ank-gradle-plugin:$VERSION_NAME"
```

Comment:
Hi @i-walker ! This change makes build-check fail because `0.10.5-SNAPSHOT` version is not found in the repositories.

However, the solution wouldn't be to add OSS repository to find that dependency because that `VERSION_NAME` will cause errors when updating the value of `VERSION_NAME` during release time (the source code cannot depend on an artifact that is deployed from that source code). 

Maybe that's the reason why that version is fixed from `0.7.1` because the Ank Gradle Plugin hasn't evolved since then. Some alternatives:

- Don't apply Arrow Ank Gradle Plugin. Add the `runAnk` task in `arrow-docs/build.gradle`.
- Exclude Arrow Ank Gradle Plugin. Change build and publish checks to control the order.